src/components/deviceprofileBuilder.ts

```diff
@@ -362,17 +363,18 @@ function getSubtitleProfiles(): Array<SubtitleProfile> {
  * @param Profile options
  * @returns Device profile.
  */
-export function getDeviceProfile(options: ProfileOptions = {}): DeviceProfile {
+export function getDeviceProfile(options: ProfileOptions): DeviceProfile {
     profileOptions = options;
     currentDeviceId = getActiveDeviceId();
 
-    const bitrateSetting = options.bitrateSetting || getMaxBitrateSupport();
-
     // MaxStaticBitrate seems to be for offline sync only
     const profile: DeviceProfile = {
-        MaxStreamingBitrate: bitrateSetting,
-        MaxStaticBitrate: 0,
-        MusicStreamingTranscodingBitrate: Math.min(bitrateSetting, 192000)
+        MaxStreamingBitrate: options.bitrateSetting,
+        MaxStaticBitrate: options.bitrateSetting,
+        MusicStreamingTranscodingBitrate: Math.min(
+            options.bitrateSetting,
+            192000
```

Comment:
is there a good reason for this 192k cap?

Reply:
1. Nah that's probably a good idea to change
2. I believe MaxStaticBitrate is leftovers from emby for syncing content but I could be wrong